store/cachekv/store.go

```diff
@@ -135,18 +135,9 @@ func (store *Store) Write() {
 	// Clear the cache using the map clearing idiom
 	// and not allocating fresh objects.
 	// Please see https://bencher.orijtech.com/perfclinic/mapclearing/
-	store.cache.Range(func(key, value any) bool {
```

Comment:
do these maps no longer behave in the idiomatic way? I gues that makes sense as we migrated from native go maps to sync maps, but just want to confirm

Reply:
Yeah, there's also a mutex to protect the writes